libpod/runtime.go

```diff
@@ -77,7 +77,7 @@ type Runtime struct {
 	ociRuntime      *OCIRuntime
 	lockDir         string
 	netPlugin       ocicni.CNIPlugin
-	ociRuntimePath  string
+	OciRuntimePath  string
```

Comment:
Would prefer an accessor function here instead of directly making this public - nothing else in runtime is public

Reply:
Fixed